tutorials/angular/start/stocks/src/app/glue.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from "@angular/core";
+import { Subject, Observable } from 'rxjs';
+

```

Comment:
Include the Client interface in type.ts
```suggestion
import {  Client } from './types';
```

Reply:
applied